app-text/capyt/capyt-1.0.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{5..6} )
+
+inherit python-r1
```

Comment:
Any reason not to use python-single-r1? Multi-impl support here doesn't seem to have any real benefit, so I would dare say it's needless added complexity.

Reply:
No real reason, I just thought one should always use multi-impl when possible! As you mentioned, it doesn't really bring any advantages in this case. Fixed below.

Reply:
Well, the answer is: it depends. I generally recommend multi-impl when you're installing Python modules for other packages to use, or when you're using distutils. Otherwise, single-impl is just simpler and easier to maintain.